src/shared/components/icon/index.tsx

```diff
@@ -1,32 +1,23 @@
 import React from 'react';
-import styled from 'styled-components';
-import { color, ColorProps, layout, LayoutProps } from 'styled-system';
 import { IconDefinition, IconSvg } from './index.generated';
-import { Theme } from '@/shared/theme';
+import cn from 'classnames';
 export * from './index.generated';
 
 type IconProps = {
     icon: IconDefinition;
     style?: React.CSSProperties;
```

Comment:
I think we can remove `style` as well since we are using classes to style things